fsharp-backend/tests/FuzzTests/OCamlInterop.FuzzTests.fs

```diff
@@ -155,7 +161,7 @@ module Roundtrippable =
     static member Dval() : Arbitrary<RT.Dval> =
       Arb.Default.Derive() |> Arb.filter (DvalReprInternal.isRoundtrippableDval true)
 
-  let roundtrip (dv : RT.Dval) : bool =
+  let canRoundtrip (dv : RT.Dval) : bool =
```

Comment:
I'm not sure "canRoundtrip" is a good name here. It implies some properties of the dval that would allow it to be roundtripped, while these functions actually do the roundtrip and report success.

Reply:
Good point - trying `roundtripsSuccessfully` - ok?